permissions/plugin-updatebot.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "updatebot"
+paths:
+- "org/jenkins-ci/plugins/updatebot"
+developers:
+- "jstrachan"
+- "rawlingsj"
```

Comment:
Account does not exist. See this repo's README for help on what's expected here.

Reply:
@daniel-beck apologies, "rawlingsj" is the github user id, on jenkins.io its "jrawlings" - fixed now!